week-2/D-predicates/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+  return false;
```

Comment:
This isn't quite right. What happens if we change `number` to be -10 instead of 5? We need to check if `number` is less than 0 instead of just hard-coding `true`. When you fix this, remember to also update the `isBetweenZeroand10` function below.